tests/language-feature/defer/switch.slang

```diff
@@ -0,0 +1,34 @@
+//TEST(compute):COMPARE_COMPUTE: -shaderobj
+//TEST(compute):COMPARE_COMPUTE: -vk -shaderobj
+//DISABLE_TEST(compute):COMPARE_COMPUTE:-cpu -shaderobj
```

Comment:
Any reason to not include the test on cpu?

Reply:
Good point, I should enable them on the CPU as well. They were left disabled because I had troubles getting CPU tests to run on my machine. I have now figured out why that is (#6734).